field.go

```diff
@@ -499,7 +508,13 @@ func (f *Field) loadMeta() error {
 
 	// Initialize "base" to "min" when upgrading from v1 BSI format.
 	if pb.BitDepth == 0 {
-		pb.Base = pb.Min
+		if pb.Min > 0 {
+			pb.Base = pb.Min
+		} else if pb.Max < 0 {
+			pb.Base = pb.Max
+		} else {
+			pb.Base = 0
+		}
```

Comment:
One minor improvement might be to make a `bsiBase(min, max uint64) uint64` that contains the logic since it's duplicated.

Reply:
good idea. i just pushed 96e6c1189793a8981938b767cbb6f1b389985362 with that change.